newIDE/app/src/AssetStore/AssetStoreSearchFilter.js

```diff
@@ -261,6 +261,155 @@ export class ColorAssetStoreSearchFilter
   }
 }
 
+// TODO
+const toAssetStoreType = (type: string) => {
+  return type === 'Sprite' ? 'sprite' : type;
+};
+
+// Thematic tags are noise for asset swapping as changing the theme may be what
+// users are looking for.
+const ignoredThematicTags = new Set([
+  'top-down',
+  'isometric',
+  'side view',
+  'pixel art',
+  'pirate',
+  'space',
+  'sea',
+  'city',
+  'medieval',
+  'nature',
+  'forest',
+]);
+
+export class AssetSwappingAssetStoreSearchFilter
+  implements SearchFilter<AssetShortHeader> {
+  isEnabled: boolean;
+  objectType: string;
+  hasSeveralState: boolean;
```

Comment:
```suggestion
  hasSeveralStates: boolean;
```

or even `hasSeveralAnimations` to be very clear, no?

Reply:
It could be only 1 frame per state for a playing card for instance.